languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,169 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PAS_DE_VIRGULE" name="pas de virgule" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="[NJVA].*" postag_regexp="yes"><exception regexp="yes">^[a-z]$</exception></token>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">et|ou</token>
+                  </marker>
+                  <token postag="[DNJV].*" postag_regexp="yes"/>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\3</suggestion>
+              <example correction="et">Elle a acheté des radis<marker>, et</marker> des choux.</example>
+              <example>Les voyelles a, e, i, o, u<marker>, ou</marker> sont couramment utilisées dans la langue française.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token>c'</token>
+                  <token skip="-1">est</token>
+                  <token>,</token>
+                  <token regexp="yes" skip="-1">qu[ei]?</token>
+                  <token regexp="yes">qu[ei]?</token>
+              </antipattern>
+              <pattern>
+                  <token>c'</token>
+                  <token skip="-1">est</token>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">qu[ei]?</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\4</suggestion>
+              <example correction="qui">C'est lui<marker>, qui</marker> part demain.</example>
+              <example correction="que">C’est l’eau laissée dans la piscine<marker>, que</marker> j'ai vidée.</example>
+          </rule>
+          <rule>
```

Comment:
This 3rd subrule has a problem — those little infos put between two commas:

> Lui-même, naturalisé depuis peu, craint d'être dénaturalisé par le gouvernement de Vichy.

We can't let LT cause alarms in those cases.

Reply:
I corrected the pattern so it should not trigger a match for this example now.